curriculum/challenges/english/25-front-end-development/quiz-javascript-object-oriented-programming/66edd5267b52f8f617d8364f.md

```diff
@@ -17,439 +17,458 @@ Answer all of the questions below correctly to pass the quiz.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Where is a restriction applied in JavaScript?
 
 #### --distractors--
 
-Placeholder distractor 1
+Class.
 
 ---
 
-Placeholder distractor 2
+Method.
 
 ---
 
-Placeholder distractor 3
+Function.
 
 #### --answer--
-
-Placeholder answer
+Data.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is TRUE?
 
 #### --distractors--
 
-Placeholder distractor 1
+Code reusability is achieved by polymorphism.
 
 ---
 
-Placeholder distractor 2
+OOP in Javascript is different from other programming languages.
 
 ---
 
-Placeholder distractor 3
+Methods and properties make up an object
 
 #### --answer--
 
-Placeholder answer
+ALL the listed.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Polymorphism achieves the following EXCEPT?
 
 #### --distractors--
 
-Placeholder distractor 1
+Method overloading.
 
 ---
 
-Placeholder distractor 2
+Method overriding.
 
 ---
 
-Placeholder distractor 3
+Changing of characteristics.
 
 #### --answer--
 
-Placeholder answer
+Binding of data.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which one of the following is TRUE about classes?
 
 #### --distractors--
 
-Placeholder distractor 1
+A class contains objects with similar properties and methods.
 
 ---
 
-Placeholder distractor 2
+Objects are instances of classes.
 
 ---
 
-Placeholder distractor 3
+Different classes can have similar methods.
 
 #### --answer--
 
-Placeholder answer
+All of the listed.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What instantiates a class?
 
 #### --distractors--
 
-Placeholder distractor 1
+Object.
 
 ---
 
-Placeholder distractor 2
+Keywords.
 
 ---
 
-Placeholder distractor 3
+Variables
 
 #### --answer--
 
-Placeholder answer
+Constructor.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Study the code below
+
+```js
+class Course
+{
+    #module;
+    constructor(module){
+      this.#module=module;
+    }
+}
+```
 
+Which access modifier is added to #module above?
 #### --distractors--
 
-Placeholder distractor 1
+Public.
 
 ---
 
-Placeholder distractor 2
+Personal.
 
 ---
 
-Placeholder distractor 3
+Global.
 
 #### --answer--
 
-Placeholder answer
+Private.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+The keyword 'super' is used to call the properties and methods of which class?
 
 #### --distractors--
 
-Placeholder distractor 1
+Child.
 
 ---
 
-Placeholder distractor 2
+Private.
 
 ---
 
-Placeholder distractor 3
+Personal.
 
 #### --answer--
 
-Placeholder answer
+Parent.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following holds true about private members?
 
 #### --distractors--
 
-Placeholder distractor 1
+It can be accessed outside the class.
 
 ---
 
-Placeholder distractor 2
+It can be called at any point in the program.
 
 ---
 
-Placeholder distractor 3
+It makes the program simpler to write.
```

Comment:
One could argue it does, though. Simple is subjective.

Reply:
Working on the raised, pertinent, issues right about now.